gulpfile.js

```diff
@@ -37,7 +37,7 @@ const { launch, connect } = require('hadouken-js-adapter');
 		logToTerminal(`kill: running: ${command}...`);
 
 		exec(command, err => {
-			if (err && !err.includes(error)) {
+			if (err && err.message && !err.message.includes(error)) {
```

Comment:
Could there be a case where err is a string instead of an Error object?

Reply:
I don't think so. Unless I am reading wrong (which does happen!) it does not look like it from the docs.

https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_child_process_exec_command_options_callback

I checked version 8 too
https://nodejs.org/docs/v0.8.28/api/child_process.html#child_process_child_process_exec_command_options_callback